src/pages/UserSettings/Teams.vue

```diff
@@ -0,0 +1,171 @@
+<script>
+import { mapActions, mapGetters } from 'vuex'
+import { clearCache } from '@/vue-apollo'
+
+import ManagementLayout from '@/layouts/ManagementLayout.vue'
+import ConfirmDialog from '@/components/ConfirmDialog'
+
+export default {
+  components: {
+    ManagementLayout,
+    ConfirmDialog
+  },
+  data() {
+    return {
+      dialogRemoveUser: false,
+      headers: [
+        { text: 'Name', value: 'name' },
+        { text: 'URL', value: 'slug' },
+        { text: 'Role', value: 'role' },
+        { text: 'Actions', value: 'id' }
+      ],
+      isRemovingUser: false,
+      removeTenant: null
+    }
+  },
```

Comment:
Could we refetch user on mount?  (I think the vuex action is getUser?) I accepted an invitation but couldn't see that team until I refreshed my page.

Reply:
<img width="620" alt="Screen Shot 2020-11-09 at 5 53 06 PM" src="https://user-images.githubusercontent.com/40272060/98606602-88887000-22b5-11eb-862a-60ea3ec4836a.png">

Reply:
4 teams in sideNav but only 3 in the teams page.

Reply:
so I've added a `getUser` and `getTenants` after accepting or declining a team invite, but its not updating the way I'd expect--specifically, I'd like to see the action icons get replaced with the relevant ones after joining a team, but the old ones just go away and are replaced with a blank space :(

Reply:
That's most likely because of https://github.com/PrefectHQ/ui/pull/403/files#diff-ceb8a586ffaa424f678e44816b6a17d8aa390c65ad1728a6cb1360570c4bbd3eR227, https://github.com/PrefectHQ/ui/pull/403/files#diff-ceb8a586ffaa424f678e44816b6a17d8aa390c65ad1728a6cb1360570c4bbd3eR242, and https://github.com/PrefectHQ/ui/pull/403/files#diff-ceb8a586ffaa424f678e44816b6a17d8aa390c65ad1728a6cb1360570c4bbd3eR257. You're referencing a computed prop by calling its getter method instead of its reactive prop - try `currentMemberships.includes(item.id)` instead.

Reply:
still not sure its working...

Reply:
I think something like `async mounted() {
    await this.getTenants()
  },` should work?

Reply:
https://blog.logrocket.com/introduction-to-vue-lifecycle-hooks/